### DISCUSSION TITLE
Hiding the Offline post.

### DISCUSSION THREAD

**Discussion by ElanHasson** [razonyang/hugo-theme-bootstrap#353] [Q&A]
How do you hide the Offline post? I can't seem to figure out how to get it not to display.

**Comment by razonyang** [comment]
Do you mean you don't need the offline page? If so, please remove the `content/offline`.

> The offline page is used by PWA.

**Comment by ElanHasson** [comment]
Hi @razonyang , thank you for the fast reply.

I want offline support for the PWA, but do not want to see a post titled "Offline", when I am not offline..

Any suggestions ?

**Reply by ElanHasson** [reply]
Here is an example: 
![image](https://user-images.githubusercontent.com/234704/150089555-9146619a-147d-4302-a514-394d93a6aba9.png)

**Comment by razonyang** [comment]
@ElanHasson Thanks for the detail. It was a bug fixed by b66754f, The related posts now only includes the `mainSections`'s posts.
